src/components/Hero.astro

```diff
@@ -0,0 +1,29 @@
+---
+import houstonOmg from "../assets/houston_omg.webp"
+---
+
+<section class="flex flex-col items-center gap-16 py-16 lg:flex-row">
+	<h1 class="heading-2 text-center lg:text-left">It’s go time with Astro</h1>
+	<div class="rounded-md bg-blue-purple-gradient p-px">
+		<section
+			class="flex flex-col gap-4 rounded-md bg-astro-gray-700 px-6 shadow-md lg:flex-row"
+		>
+			<div class="flex flex-1 flex-col gap-4 py-6 text-astro-gray-200">
+				<h2 class="font-bold text-astro-gray-100">Let’s get started</h2>
+				<p>
+					Ready to get started with an Astro project but want to get to building
+					right away instead of spend time setting up your local environment?
+				</p>
+				<p>
+					Open any of these templates in your choice of online code editor right
+					in your browser, or view the source code in the GitHub repo.
+				</p>
+			</div>
+			<img
+				src={houstonOmg}
+				alt="Astro's mascot, Houston, making an 'omg' face"
+				class="self-center lg:self-end"
+			/>
```

Comment:
Seeing a little bit of layout shift that I think is due to no `width` & `height` on here? Same applies to the Houston at the page bottom, but most crucial here.

```suggestion
			<img
				src={houstonOmg}
				alt="Astro's mascot, Houston, making an 'omg' face"
				class="self-center lg:self-end"
				width="156"
				height="104"
			/>
```